### ISSUE TITLE
ActionController::RoutingError presents a 500 error

### ISSUE THREAD

**Issue by artero** [MarsBased/pathfinder-rails#89]
ActionController::RoutingError error should present 404,

https://github.com/MarsBased/pathfinder-rails/blob/7f74a216bce575b1aa7288fac0fb86d00776bb99/recipes/configuration.rb#L76

Probably the problem is in the routes:
https://gist.github.com/joneslee85/f2b4565949f7c988c9481b3c5dc15dba

**Comment by oriolbcn** [comment]
I think the problems is in that the errors controllers can't render the views because they don't exist

**Comment by xredo** [comment]
I agree with Oriol. If we add this error handling code we need to add a default view for 404 and 500 errors.

**Comment by artero** [comment]
Yes, I tested and the problem is because the views are not created. I agree with you, the views should be created or we should delete this method.
